### ISSUE TITLE
NEOS-112: Improve the Edit / Preview tab UI

### ISSUE THREAD

**Issue by johannessteu** [johannessteu/neos-development-collection#260]
Jira issue originally created by user rasmusskjoldan:

If a user has a Preview mode selected, collapses the Edit / Preview panel, tries to edit content but is not aware of the Preview mode being selected, there is insufficient UI feedback.

It might be needed that we need to do more tasks to solve this but to start with we need to introduce the following changes:

• Use 2 different colors for both Edit / Preview labels and the mode buttons
• All Edit mode buttons keep the same primary color 
• All Preview mode buttons should be green (#ff8700)
• Mark the active Edit / Preview category corresponding to the selected mode: If a Preview mode is selected and the Edit / Preview panel is collapsed, then the label "Preview" in the tab is both bold and #ff8700. If an Edit mode is selected the "Edit" label is both bold and blue

See attached designs.

 Atachements:

https://jira.neos.io/secure/attachment/10203/Screen+Shot+2014-04-23+at+15.56.18.png 

https://jira.neos.io/secure/attachment/10202/Screen+Shot+2014-04-23+at+15.56.36.png 

https://jira.neos.io/secure/attachment/10201/Screen+Shot+2014-04-23+at+15.57.20.png 

 Jira-URL: https://jira.neos.io/browse/NEOS-112

**Comment by johannessteu** [comment]
Comment created by christopher:

Shouldn't we also remove the inspector in that view? You shouldn't be able to edit properties of the current document anyway.

Just an idea for something more obvious: have a sensitive area in the beginning of the context bar with a return icon to "switch back" to the last edit mode. But I think we need that space for the dimensions selector and maybe some other stuff affecting the preview (e.g. responsive viewport selector).

**Comment by johannessteu** [comment]
Comment created by rasmusskjoldan:

About the inspector: Probably yes but we need to think it through in regard to how it will work also after we've opened up for tabs in the inspector... I'm not totally sure it will work to remove it. What about just collapsing it when selecting a preview mode?

About switching back: I'm not totally sure I follow you there – what's the typical use case? 

Shortcuts for Edit / Preview modes would be nice, by the way...

**Comment by johannessteu** [comment]
Comment created by christopher:

Regarding switching back: I noticed that when I did some preview-heavy editing of content I needed to go back and forth between "In Place Editing" and "Desktop" quite often. If you imagine multiple editing and preview modes that could get annoying. It's nothing dramatic but keeping the last selected mode in the preview or edit section could help (if there's a way to switch between the two).
